tools/BuildGraph/DGMLFormatter.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+
+namespace BuildGraph
+{
+    public class DGMLFormatter : GraphFormatter
```

Comment:
I'd take a look at the code. BuildTrain didn't really get any eyes because it was very prototypey and then kinda just got abandoned :)